.github/workflows/check.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         run: |
           bundle exec rspec
           bundle exec rspec spec/slow/*_spec.rb
+        env:
+          RUBYOPT: "--enable-frozen-string-literal --debug-frozen-string-literal"
```

Comment:
NB: only enabling it for `rspec` because `brakeman` isn't yet compatible.

Reply:
Could you add a comment about this, maybe link to brakeman's tracking issue? I'm never going to remember why there's this distinction and #1286 has some upcoming edits to this file.

Reply:
I haven't opened an issue on brakeman yet, they have a couple dependencies that need a fix first. But I'll open it.

Reply:
Comment added.